security/pam-reattach/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+name                pam-reattach
+github.setup        fabianishere pam_reattach 1.2 v
+revision            0
+
+categories          security
+license             MIT
+maintainers         {@kakuhen} openmaintainer
+platforms           darwin
+supported_archs     x86_64 arm64
+description         PAM module to reattach to user's per-session bootstrap namespace
+long_description    \
+    A PAM module for reattaching to the authenticating user's per-session bootstrap \
+    namespace on macOS. This allows users to make use of the pam_tid module within \
+    programs like screen and tmux.
+
+checksums \
+    sha256    4e2b576c03f9ff6f8a4f02d750540f32cf9529b3cf6cd6fb22f5df0765a5a27a \
+    rmd160    fed5f6266e8dced66223c4c00778894b368cd300 \
+    size      6218
```

Comment:
see above, after changing to the `archive` on GitHub please update and keep the alignment the same as for the rest of the Portfile (i.e., no reason for a newline)